R/rsync/build_tarballs.jl

```diff
@@ -40,6 +41,9 @@ make install
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = supported_platforms()
+# xxHash is not yet available for riscv64 or aarch64-*-freebsd
+filter!(p -> arch(p) != "riscv64", platforms)
+filter!(p -> !(Sys.isfreebsd(p) && arch(p) == "aarch64"), platforms)
```

Comment:
xxHash should be available there in a short while, since https://github.com/JuliaPackaging/Yggdrasil/pull/10271 was just merged.

Reply:
Yes it is.